docs/development/affiliated-packages.rst

```diff
@@ -282,17 +282,15 @@ files manually`_ section since this explains what many of the files do.
    license.
 
 #. Take a moment to look over the ``<packagename>/example_mod.py``,
-   ``<packagename>/tests/test_example.py``, ``scripts/script_example``,
-   and ``<packagename>/example_c.pyx`` files, as well as the
-   ``<packagename>/example_subpkg`` directory. These are examples of a
-   pure-python module, a test script, an example command-line script, a
+   ``<packagename>/tests/test_example.py``, and ``<packagename>/example_c.pyx``
+   files, as well as the ``<packagename>/example_subpkg`` directory.
+   These are examples of a pure-python module, a test script, a
```

Comment:
Apparently, I didn't read it carefully enough. I think you're right. Please ignore the noise!